demo/tab/basic/playground

```diff
@@ -0,0 +1,20 @@
+const tabs = [
+  'Vincent',
+  'Jules',
+  'Marcellus',
+  'Mia',
+  'Winston',
+  'Butch'
```

Comment:
Truth is I was thinking to even use a button instead an anchor... :D I don't see the point to keep the `url` functionality anymore otherwise I should even consider React Router Links and I don't see that use right now.